### PR TITLE
RA-1618 : Add Head circumference Concept to the default concepts in reff app

### PR DESCRIPTION
**Pull request by mozzy11**
Ticket :https://issues.openmrs.org/browse/RA-1618 

Add Head circumference Concept to the default concepts needed by Reff app For Growth charts

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you see the travis build failure?

**Comment by mozzy11** [comment]
@dkayiwa i corrected the Pr. it was a matter of switching from oracle jdk to openjdk

File: api/src/main/resources/Reference_Application_Concepts-24.xml [hunk 1]
```diff
@@ -116,6 +116,7 @@
   <concept concept_id="162822" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:33:22.0" version="" changed_by="1" date_changed="2015-01-06 22:33:29.0" retired_by="1" uuid="162822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="163139" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-12-13 15:00:18.0" version="" changed_by="1" date_changed="2015-12-13 15:02:03.0" retired_by="1" uuid="163139AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="165095" retired="false" datatype_id="3" class_id="7" is_set="false" creator="1" date_created="2018-04-18 14:34:20.0" version="" changed_by="1" date_changed="2018-04-18 14:34:30.0" retired_by="1" uuid="165095AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="5314" retired="false" datatype_id="1" class_id="5" is_set="false" creator="1" date_created="2016-11-11 14:34:20.0" version="" changed_by="1" date_changed="2018-04-18 14:34:30.0" retired_by="1" uuid="5314AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```
Comment (dkayiwa) [inline api/src/main/resources/Reference_Application_Concepts-24.xml]: Why should we have a retired_by value when the concept is not retired?

Comment (mozzy11) [reply api/src/main/resources/Reference_Application_Concepts-24.xml]: well, yourright but that seemstobe the general way right from above

**Pull request merged**